src/Database/Type/IntegerType.php

```diff
@@ -101,6 +101,14 @@ public function manyToPHP(array $values, array $fields, Driver $driver)
             if (!isset($values[$field])) {
                 continue;
             }
+
+            if ($values[$field] !== null && $values[$field] !== '' && !is_numeric($values[$field])) {
```

Comment:
* no need for `$values[$field] !== null` there is an `isset` check for the value a few lines above

* no need for `$values[$field] !== ''` the `is_numeric`, we want the `is_numeric` check to catch it

Reply:
:+1: removed them